.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: regentmarkets/debian-pg:stretch-5.26.2-96
```

Comment:
Why `-pg`? This should only be using Perl, I'd expect `regentmarkets/debian-perl:stretch-5.26.2` here.

Reply:
I copied from another place :p